jobs/release/release_debian.groovy

```diff
@@ -6,16 +6,18 @@ node(build_debian_node){
         "BINTRAY_DISTRIBUTION=trusty", 
         "BINTRAY_ARCHITECTURE=amd64"]){
         deleteDir()
-        dir("on-build-config"){
-            checkout scm
-        }
-        dir("DEBIAN"){
-            unstash "debians"
-        }
-        withCredentials([
-            usernameColonPassword(credentialsId: 'a94afe79-82f5-495a-877c-183567c51e0b', 
-                                  variable: 'BINTRAY_CREDS')]){
-            sh './on-build-config/jobs/release/release_debian.sh'
+        if (env.NEW_DEB == true ){
```

Comment:
thanks to @changev's explanation.
the NEW_DEB is an improvement flag.
if the deb package has been already cached in the artifactory/bintray, then we can skip the publish step.